de4dot.code/deobfuscators/MethodStack.cs

```diff
@@ -91,7 +79,8 @@ public static class MethodStack {
 					break;
 				if (instr.OpCode.Code == Code.Dup) {
 					pushes = 1;
-					pops = 0;
+					pops = 0;
+					instr = GetPushedArgInstructions(instructions, index, 1).GetEnd(0) ?? instr;
```

Comment:
This code seems to cause an infinite loop. Try deobfuscating .NET Reactor 4.7.0.0

Reply:
Corrected but have no file to test